6thShopProject2025/src/main/java/frc/robot/subsystems/BeltSubsystem.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.CANSparkMax;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class BeltSubsystem extends SubsystemBase {
+
+  // declaring each motor
+  private final WPI_TalonSRX mainBeltMotor;
+
+  /** Creates a new ExampleSubsystem. */
+  public BeltSubsystem() {
+
+    mainBeltMotor = new WPI_TalonSRX(3);
+    mainBeltMotor.setNeutralMode(NeutralMode.Coast); // turning braking to Coast
```

Comment:
You probably want the roller to be on brake. Otherwise the gamepiece might slip out.